config/linux-x230.config

```diff
@@ -218,7 +214,6 @@ CONFIG_MSDOS_FS=y
 CONFIG_VFAT_FS=y
 # CONFIG_PROC_SYSCTL is not set
 # CONFIG_PROC_PAGE_MONITOR is not set
-CONFIG_TMPFS=y
```

Comment:
This one as well.  While it makes sense to use ramfs (instead of tmpfs) in an environment that will never have swap, this should be a separate PR.